add hover effects on "obtain line" and "cert. of analysis"

rugeli opened the pull request:
Problem
=======
What is the problem this work solves, including
closes #27

Solution
========
What I/we did to solve this problem

- modified css to apply hover effects at the cell level for the last two columns 

Note: 

- The color toggle for the three little vertical strokes within the `Tube` icon isn't working. Should we make adjustments to the css or the svg?

- I noticed that our cert svg slightly differs from the one in the design, not sure if this is significant


## Type of change
Please delete options that are not relevant.
* New feature (non-breaking change which adds functionality)

Screenshots (optional):
-----------------------
<img width="1667" alt="Screenshot 2024-03-14 at 2 53 34 PM" src="https://github.com/allen-cell-animated/cell-catalog/assets/91452427/2cad6348-9d72-4480-aef5-dfee1ce9ed8d">



The `Cert.` icon in the design:
 
<img width="163" alt="Screenshot 2024-03-14 at 2 36 01 PM" src="https://github.com/allen-cell-animated/cell-catalog/assets/91452427/962b946b-cb91-4ce9-bd85-f914bcfc5a78">

meganrm commented:
@rugeli I made one commit where I changed the selector to use a class name instead of the column number. This is more future proof because if we added or removed a column from the table, the selector would have gotten messed up

rugeli commented:
> @rugeli I made one commit where I changed the selector to use a class name instead of the column number. This is more future proof because if we added or removed a column from the table, the selector would have gotten messed up. I also made it so the cell didn't get a hover effect if there wasn't a link

These are great changes to prevent potential issues!
